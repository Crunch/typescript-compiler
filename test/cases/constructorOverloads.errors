test/cases/constructorOverloads.ts(3,5): error TS2392: Multiple constructor implementations are not allowed.
