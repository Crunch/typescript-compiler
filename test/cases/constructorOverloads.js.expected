var C = (function () {
    function C(x) {
    }
    return C;
})();
var D = (function () {
    function D(x) {
    }
    return D;
})();
